spec/03-plugins/09-key-auth/02-access_spec.lua

```diff
@@ -405,6 +419,25 @@ for _, strategy in helpers.each_strategy() do
       end)
     end)
 
+    describe("key in gRPC headers", function()
+      it("rejects call without credentials", function()
+        assert.falsy(helpers.proxy_client_grpc(){
```

Comment:
Could we add an assertion for the returned code / message as well, here and in other plugins tests?

Reply:
Can be done in a separate PR.

Reply:
some plugins use JSON for the body. even if the status is appropriate (401 or similar), grpcurl can't give a readable error.  I don't think hardcoding the `Message: transport: received the unexpected content-type "application/json; charset=utf-8"` message is appropriate.  also, it's generated by grpcurl, and it could change.